README.md

```diff
@@ -2,83 +2,40 @@
 
 [![Build Status](https://travis-ci.org/kubernetes-incubator/service-catalog.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/service-catalog)
 
-### Terminology
+### Introduction
 
-**Service**: running code that is made available for use by an application.
-Traditionally, services are available via HTTP REST endpoints, but this is not
-a requirement.
+The service-catalog project is in incubation to bring integration with service
+brokers to the Kubernetes ecosystem.  A service broker is an endpoint that
+manages a set of services.  The end-goal of the service-catalog project is to
+provide a way for Kubernetes users to consume services from brokers and easily
+configure their applications to use those services.
 
-**Service Broker**: An endpoint that manages a set of services. Responsible for
-translating Service Catalog activities (like provision, bind, unbind, deprovision)
-into appropriate actions for the service.
+<hr>
```

Comment:
yeah, I don't know how to do an hr in markdown, do you?  I was following the format of the Kubernetes readme.